docs-source/content/userguide/managing-domains/accessing-the-domain/admin-console.md

```diff
@@ -1,6 +1,90 @@
+
+For detailed documentation, see the [Oracle WebLogic Server Remote Console](https://github.com/oracle/weblogic-remote-console) GitHub project.
+
+A major benefit of using the Remote Console is that you don't need to install or run the WebLogic Server Administration Console on WebLogic Server instances.
+You can use the Remote Console with WebLogic Server **slim** installers, available on the [OTN](https://www.oracle.com/middleware/technologies/weblogic-server-installers-downloads.html)
+or [OSDC](https://edelivery.oracle.com/osdc/faces/Home.jspx;jsessionid=LchBX6sgzwv5MwSaamMxrIIk-etWJLb0IyCet9mcnqAYnINXvWzi!-1201085350).
+Slim installers reduce the size of WebLogic Server downloads, installations, container images, and Kubernetes pods.
+For example, a WebLogic Server 12.2.1.4 slim installer download is approximately 180 MB smaller.
+
+## Use the Remote Console
+
+To access WebLogic Server domains running in Kubernetes:
+
+1. Install and configure the Remote Console according to these [instructions](https://github.com/oracle/weblogic-remote-console/blob/master/site/install_config.md).
+
+1. For the Remote Console to access the Administration Server running in Kubernetes, you can:
+
+   * Use [curl](#use-curl).
+   * Use the [Administration Server `NodePort`](#use-the-administration-server-nodeport).
+   * Configure [ingress path routing rules](#configure-ingress-path-routing-rules).
+
+
+### Use curl
+
+Access the REST interface of the WebLogic Server Administration Server to verify the connection and that the correct `hostname:port` is being used:
+
+```
+$ curl --user username:password http://${HOSTNAME}:${LB_PORT}/console/login/LoginForm.jsp
+```
+
+* `${HOSTNAME}` is where you start up the WebLogic domain.
+
+* To determine `${LB_PORT}`:
+
+   `$ export LB_PORT=$(kubectl -n traefik get service traefik-operator -o jsonpath='{.spec.ports[?(@.name=="web")].nodePort}')`
+
+
+### Use the Administration Server `NodePort`
+
+For the Remote Console to connect to the Kubernetes WebLogic Server Administration Server’s `NodePort`:
+
+```
+http://hostname:adminserver-NodePort/
+```
+
+### Configure ingress path routing rules
+
+1. Configure an ingress path routing rule. For example, see the following `path-routing` YAML file.
+
+   ```yaml
+   apiVersion: traefik.containo.us/v1alpha1
+   kind: IngressRoute
+   metadata:
+     annotations:
+       kubernetes.io/ingress.class: traefik
+     name: traefik-pathrouting-1
+     namespace: weblogic-domain
+   spec:
+     routes:
+     - kind: Rule
+       match: PathPrefix(`/domain1`)
+       services:
+       - kind: Service
+         name: domain1-cluster-dockercluster
+         namespace: weblogic-domain
+         port: 8001
+     - kind: Rule
+       match: PathPrefix(`/`)
+       services:
+       - kind: Service
+         name: domain1-adminserver
+         namespace: weblogic-domain
+         port: 7001
+   ```
+
+1. To connect to the Remote Console:
```

Comment:
For the Remote Console to connect to the Kubernetes WebLogic Server Administration Server use the URL

Reply:
I am not sure the LBPORT is necessary when using path routing

Reply:
I believe that I got that info from Maggie's JIRA, https://jira.oraclecorp.com/jira/browse/OWLS-85541. Perhaps, she can confirm or edit the URL, @maggiehe00?

Reply:
@mriccell @maggiehe00 said, "Yes, we do need LBPORT."

Reply:
Yes, we still need LBPORT for path routing

Reply:
@tbarnes-us Why would you want to run the remote console backend in a pod inside of K8s (internal approach) we do not even distribute images of the remote console. What are the benefits?
Is "on-premise" the right term?
I think you mean by on-premise when you run the remote console backend on your local desktop, is that correct? The term on-premise is overloaded and is not clear.

Reply:
Tom's comment and distinction between the two URLs is what I am aiming to clarify in this doc.   There are two URLs 

- One you enter in the browser to start the Remote Console Front  End,  often 'http://localhost:8012' (refer to  https://github.com/oracle/weblogic-remote-console/blob/master/site/install_config.md#start-connect)
- Second is the URL required so that the Remote Console can connect to the WebLogic Domain running in Kubernetes, refer to https://github.com/oracle/weblogic-remote-console/blob/master/site/install_config.md#SSL
The first URL is well documented in the Remote Console documentation and we can simply point to those docs.  It is important that we  document how to construct the second URL when we have configured LB path routing rules.